doc/ref/states/backup_mode.rst

```diff
@@ -24,6 +24,14 @@ Or it can be set for each file:
         - source: salt://ssh/sshd_config
         - backup: minion
 
+
+The backup_mode can be set to any of thefollowing settings:
+
+* minion: backup to the minion
+* master: backup to the master, not yet implimented.
+* both: backup to both. currently only works the sam as minion.
```

Comment:
**Suggestion (non-blocking, phrasing):** It wasn't entirely clear to me when I first read this what was being backed up until I read the text before this. Maybe just for clarity, let's add a little more to the option descriptions and also make the options either format them in bold or double-backticks to set them apart:

* **minion:** Creates a backup of the file that is stored on the minion.
* **master:** Creates a backup of the file that is stored on the master, but is not yet implemented.
* **both:** Creates a backup of the file that is stored on both the master and the minion. Currently, this option only works the same as selecting the minion option.

**Question:** I don't know what it "not yet implemented" means in this context. I also don't know where the backup gets stored. I also don't know what that means that it operates the same as the minion. Can you provide some more clarity?

**Nitpick (blocking, spelling):** FYI, "implemented" and "same" were misspelled in the original text. There were also some grammar issues. So if you don't use my edits, at least fix those mistakes.

Reply:
as the backup location was described in the very next section. so didn't think it was necessary to rehash that. correcting everything else.